bindings_wasm/src/conversations.rs

```diff
@@ -641,4 +641,16 @@ impl Conversations {
     );
     Ok(StreamCloser::new(stream_closer))
   }
+
+  #[wasm_bindgen(js_name = "streamConsent")]
+  pub fn stream_consent(&self, callback: StreamCallback) -> Result<StreamCloser, JsError> {
```

Comment:
This looks about right, what's not working ?

Reply:
see the clippy issue in CI